src/main/java/net/openhft/chronicle/wire/ValueOut.java

```diff
@@ -481,7 +484,11 @@ default WireOut typedMarshallable(@Nullable Serializable object) {
     }
 
     default boolean isScalar(Serializable object) {
-        return object instanceof Comparable;
+        if (object instanceof Comparable) {
```

Comment:
you need to rebase I think - this change has already been made to develop

Reply:
Rebase performed